docs/install.sh

```diff
@@ -0,0 +1,8 @@
+# Copy over poetry dependencies from theming repository
+cp pytket-docs-theming/extensions/pyproject.toml .
+cp pytket-docs-theming/extensions/poetry.lock .
+
+# Install the docs dependencies. Creates a .venv directory in docs
+poetry install
+
+# NOTE: Editable wheel should be installed separately.
```

Comment:
Do you think the editable wheel should be installed by the `install.sh` script as well? Or should that be left to the user?

Reply:
If by this you refer to running `pip install .` on the local version of the repo, I think it's better to ask the user to do it separately.

Reply:
I think I agree